tests/unit/fem/test_quadinterpolator.cpp

```diff
@@ -194,11 +240,11 @@ static bool testQuadratureInterpolator(const int dim,
 TEST_CASE("QuadratureInterpolator", "[QuadratureInterpolator]")
 {
    const auto d = GENERATE(2,3); // dimension
-   const auto p = GENERATE(range(1,8)); // element order
-   const auto q = GENERATE_COPY(range(p+1,10)); // quadrature order
+   const auto p = GENERATE(range(1,7)); // element order, 1 <= p < 7
+   const auto q = GENERATE_COPY(p+1,p+2); // 1D quadrature points
    const auto l = GENERATE(QVectorLayout::byNODES, QVectorLayout::byVDIM);
-   const auto nx = GENERATE(3,4); // number of element in x
-   const auto ny = GENERATE(5,6); // number of element in y
-   const auto nz = GENERATE(1,3); // number of element in z
-   REQUIRE(testQuadratureInterpolator(d, p, q, l, nx, ny, nz));
+   const auto nx = 3; // number of element in x
```

Comment:
Forcing only `3` here won't allow a latitude in the inner part of the domain for the randomized tested vectors.

Reply:
All elements are still deformed, even with the fixed boundary. We can also perturb the boundary if we want -- no need to fix it in this test.